verification.py

```diff
@@ -377,205 +392,158 @@ def verify_recombination(
         pyplot.close("all")
 
 
-class MsTest(SimulationVerifier):
-    def __init__(self, output_dir):
-        super().__init__(output_dir)
+class MsCmd1(Test):
```

Comment:
I think it would be better to see these as Test classes, so we can call this ``MsTest``. Same for the other classes below.